R/Model-class.R

```diff
@@ -160,6 +160,8 @@ ModelLogNormal <- function(mean, cov, ref_dose = 1) {
 #' [`LogisticNormal`] is the class for the usual logistic regression model with
 #' a bivariate normal prior on the intercept and slope.
 #'
+#' Typically, end users will not use the `.DefaultLogisticNormal()` function.
```

Comment:
```suggestion
#' @note Typically, end users will not use the `.DefaultLogisticNormal()` function.
```
and I would put this directly on top of line 216 (since then again it is easier to maintain), and then similar for the other constructor documentation